plugins/notifications/email/email.yaml

```diff
@@ -0,0 +1,35 @@
+# Don't change this
+type: email
+
+name: email_default # this must match with the registered plugin in the profile
+log_level: info # Options include: trace, debug, info, warn, error, off
+
+format: |  # This template receives list of models.Alert objects
+  {{range . -}}
+    {{$alert := . -}}
+    {{range .Decisions -}}
+      <a href=https://www.whois.com/whois/{{.Value}}>{{.Value}}</a> will get <b>{{.Type}}</b> for next <b>{{.Duration}}</b> for triggering <b>{{.Scenario}}</b>. <a href=https://www.shodan.io/host/{{.Value}}>Shodan</a>  
+    {{end -}}
+  {{end -}}
+
+smtp_host: # eg value smtp.gmail.com
+smtp_username:  #Replace this with your actual username
+smtp_password:  #Replace this with your actual password
+smtp_port: # Common values are any of [25, 465, 587, 2525]
+auth_type: # Valid choices are either of "none", "crammd5", "login", "plain" 
+sender_email: # eg: foo@gmail.com
+email_subject: CrowdSec Notification
```

Comment:
is this a go template too ?

Reply:
no, the email subject would be static after the sets it in config